### DISCUSSION TITLE
vue-gapi relationship

### DISCUSSION THREAD

**Discussion by namoscato** [wavezync/vue3-google-signin#6] [Ideas]
👋🏻 @kasvith!

I have helped maintain [vue-gapi](https://github.com/vue-gapi/vue-gapi) over the years, and your plugin was [brought to my attention](https://github.com/vue-gapi/vue-gapi/issues/315#issuecomment-1191369276) as we do not currently support Google Identity Services.

My availability is becoming limited, so I would certainly be open to working with you to bridge these projects via any number of options, including:

* deprecating `vue-gapi` and pointing people here
* moving these projects under a generalized GitHub organization
* extracting some core framework-agnostic library (that could be used across Vue, React, etc)

Let me know if you have any thoughts / preference!

Also looping in @fmeyertoens and @cedpoilly who have helped with `vue-gapi` as well 😄

**Comment by kasvith** [comment]
Hi @namoscato 

Thanks for reaching out. And also thanks for maintaining such an excellent library for so long. 🙌 

If you prefer, you can point users to this library, we would love that ❤️ LMK any thoughts
Also, contributions from everyone are welcome 💯 

Regarding the last two points, I would like to think about a framework-agnostic method to perform it. Currently, we are using Vue3 Reactivity heavily with this version of the library.

**Reply by namoscato** [reply]
Thanks, @kasvith!

[v2.1.2](https://github.com/vue-gapi/vue-gapi/releases/tag/v2.1.2) adds a [deprecation notice](https://github.com/vue-gapi/vue-gapi#%EF%B8%8F-deprecation) pointing here.

**Reply by kasvith** [reply]
Thank you :)
